tests/index.html

```diff
@@ -1,33 +1,36 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta charset="utf-8" />
```

Comment:
Don't think the trailing slashes are required, I'd not add them

Reply:
Sorry, my vscode added them automatically. I will remove them

Reply:
thank you!